tests/e2e/matmul/generate_e2e_matmul_tests.py

```diff
@@ -251,26 +251,26 @@ def get_rocm_test_compilation_infos(compilation_info_id: CompilationInfoId):
     assert compilation_info_id == CompilationInfoId.LLVMGPUVectorDistribute
 
     schedules = [
-        MMASchedule("F16_16x16x16_F32", 1, 1, 1, 1, 1),
-        MMASchedule("F16_16x16x16_F32", 1, 1, 1, 1, 2),
-        MMASchedule("F16_16x16x16_F32", 1, 1, 1, 2, 1),
-        MMASchedule("F16_16x16x16_F32", 1, 1, 2, 1, 1),
-        MMASchedule("F16_16x16x16_F32", 2, 2, 1, 1, 1),
-        MMASchedule("F16_16x16x16_F32", 2, 4, 2, 1, 2),
-        MMASchedule("F16_16x16x16_F32", 4, 2, 4, 2, 2),
-        MMASchedule("F16_32x32x8_F32", 1, 1, 1, 2, 2),
-        MMASchedule("F16_32x32x8_F32", 2, 2, 1, 1, 1),
-        MMASchedule("F16_32x32x8_F32", 1, 4, 2, 1, 2),
-        MMASchedule("F16_32x32x8_F32", 4, 2, 1, 2, 4),
+        MMASchedule("MFMA_F16_16x16x16_F32", 1, 1, 1, 1, 1),
+        MMASchedule("MFMA_F16_16x16x16_F32", 1, 1, 1, 1, 2),
+        MMASchedule("MFMA_F16_16x16x16_F32", 1, 1, 1, 2, 1),
+        MMASchedule("MFMA_F16_16x16x16_F32", 1, 1, 2, 1, 1),
+        MMASchedule("MFMA_F16_16x16x16_F32", 2, 2, 1, 1, 1),
+        MMASchedule("MFMA_F16_16x16x16_F32", 2, 4, 2, 1, 2),
+        MMASchedule("MFMA_F16_16x16x16_F32", 4, 2, 4, 2, 2),
+        MMASchedule("MFMA_F16_32x32x8_F32", 1, 1, 1, 2, 2),
+        MMASchedule("MFMA_F16_32x32x8_F32", 2, 2, 1, 1, 1),
+        MMASchedule("MFMA_F16_32x32x8_F32", 1, 4, 2, 1, 2),
+        MMASchedule("MFMA_F16_32x32x8_F32", 4, 2, 1, 2, 4),
```

Comment:
Would be good to add tests for WMMA here also at some point. A TODO for now would be good enough.

Reply:
sgsg